homeassistant/components/knx/schema.py

```diff
@@ -30,13 +30,13 @@
 ##################
 
 ga_validator = vol.Any(
-    cv.matches_regex(GroupAddress.ADDRESS_RE),
+    cv.matches_regex(GroupAddress.ADDRESS_RE.pattern),
```

Comment:
ADDRESS_RE is a compiled regex. Compiling this twice worked well, but doesn't type check.